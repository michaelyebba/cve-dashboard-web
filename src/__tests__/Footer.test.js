import React from 'react';
import {HashRouter} from 'react-router-dom'
import renderer from 'react-test-renderer';
import CveDetailPage from '../components/pages/Cvedetail';

it('renders correctly', () => {
  const tree = renderer
    .create(
        <HashRouter>
            <CveDetailPage/>
        </HashRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});