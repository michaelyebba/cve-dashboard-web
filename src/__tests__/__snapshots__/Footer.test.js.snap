// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <header
    id="header"
  >
    <div
      className="container"
    >
      <nav>
        <a
          className="logo"
          href="#/"
          onClick={[Function]}
        >
          <i
            className="fas fa-exclamation-triangle"
          />
          TOP 10 CVE Dashboard
        </a>
        <ul
          className="reports"
        >
          <li>
            <a
              aria-current="page"
              className="active"
              href="#/"
              onClick={[Function]}
              style={Object {}}
            >
              ALL
            </a>
          </li>
          <li>
            <a
              aria-current={null}
              href="#/cves/severity/HIGH"
              onClick={[Function]}
            >
              HIGH
            </a>
          </li>
          <li>
            <a
              aria-current={null}
              href="#/cves/severity/MEDIUM"
              onClick={[Function]}
            >
              MEDIUM
            </a>
          </li>
          <li>
            <a
              aria-current={null}
              href="#/cves/severity/LOW"
              onClick={[Function]}
            >
              LOW
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </header>
  <section
    id="report-section"
  >
    <div
      className="container"
    >
      <div
        className="report-title"
      >
        <p
          className="h3"
        >
          CVE Detail
        </p>
      </div>
      <div>
        <table
          className="table table-sm"
        >
          <thead>
            <tr>
              <th>
                Key
              </th>
              <th>
                Value
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>
                ID
              </td>
              <td />
            </tr>
            <tr>
              <td>
                Date
              </td>
              <td />
            </tr>
            <tr>
              <td>
                Severity
              </td>
              <td />
            </tr>
            <tr>
              <td>
                Description
              </td>
              <td />
            </tr>
            <tr>
              <td>
                Category
              </td>
              <td />
            </tr>
            <tr>
              <td>
                Products(s)
              </td>
              <td />
            </tr>
          </tbody>
        </table>
      </div>
    </div>
  </section>
  <footer
    id="footer"
  >
    <div
      className="container"
    >
      <p>
        <small>
          Common Vulnerabilities and Exposures (CVE) is a catalog of known security threats. The catalog is sponsored by the United States Department of Homeland Security (DHS). 
        </small>
      </p>
    </div>
  </footer>
</div>
`;
